lib/ansible/cli/playbook.py

```diff
@@ -162,7 +162,7 @@ def _process_block(b):
                                 if isinstance(task, Block):
                                     taskmsg += _process_block(task)
                                 else:
-                                    if task.action == 'meta':
+                                    if task.action == 'meta' and task.implicit is True:
```

Comment:
Just curious, why the strict `is True` check here?

Reply:
dunno, oversight during many edits